src/openrct2/rct1.h

```diff
@@ -26,8 +26,9 @@
 #include "world/map.h"
 #include "world/sprite.h"
 
-#define RCT1_MAX_MAP_ELEMENTS   0xC000
-#define RCT1_MAX_SPRITES        5000
+#define RCT1_MAX_MAP_ELEMENTS       0xC000
```

Comment:
Why are these aligned using spaces? Only seem to be two cases of spaces used in this file